src/openms/include/OpenMS/ANALYSIS/TOPDOWN/DeconvolvedSpectrum.h

```diff
@@ -140,8 +137,11 @@ namespace OpenMS
     /// set precursor peakGroup
     void setPrecursorPeakGroup(const PeakGroup& pg);
 
-    /// set precursor Qvalue
-    void setPrecursorQvalue(float q, PeakGroup::DummyIndex flag);
+    /** set precursor Q value. Q value can be retrieved by getPrecursorPeakGroup().GetQvalue()
+     * @param q Q value
+     * @param dummy_index  This index specifies if a PeakGroup is a target (0), charge dummy (1), noise dummy (2), or isotope dummy (3)
+     */
+    void setPrecursorQvalue(float q, PeakGroup::TargetDummyType dummy_index);
```

Comment:
it would be slightly cleaner to delegate with getPrecursorPeakGroup().setQvalue(...)
to make it consistent

Reply:
done